tools/create-cluster

```diff
@@ -42,16 +41,12 @@ networking:
     hostSubnetLength: 9
   machineCIDR: 192.168.126.0/24
   serviceCIDR: 172.30.0.0/16
-  type: OpenshiftSDN
+  type: OpenShiftSDN
```

Comment:
You could also just omit this and let the install pick the default, which is `OpenShiftSDN`.

You could also omit and use defaults for `networking.clusterNetworks`, `networking.serviceCIDR`, `platform.libvirt.URI`, and `platform.network`.

Also, you don't need `compute[].platform` or `controlPlane.platform`.

Reply:
:+1: thanks will do that!